C-Plus-Plus/cp/prime_factors.cpp

```diff
@@ -0,0 +1,54 @@
+/* This algorithm will represent a number n(>1) as product of prime numbers.
+ For example, if the input is 12 then output will be 2 2 3 , if input is 28 then output will be (2,2,7).
+ Two concepts are used here. Firstly , through sieve of erathosthenes we will find all the prime numbers till n.
+ Then we will check which prime numbers can divide n and for how many times*/
```

Comment:
@divya-rai-42 Provide a line break here.

Reply:
Sir, I have done the changes. I request you to review it again.